contrib/babelfishpg_common/src/collation.c

```diff
@@ -1517,4 +1517,27 @@ babelfish_update_server_collation_name(PG_FUNCTION_ARGS)
 	server_collation_name = pstrdup(babelfish_restored_server_collation_name);
 	MemoryContextSwitchTo(oldContext);
 	PG_RETURN_VOID();
+}
+
+
+/*
+ * print_const_collation - determines whether collation needs to be dumped or not
+ * for collatable data types such as varchar, char, nvarchar, nchar, etc.
+ */
+bool
+print_const_collation(Const *constval) {
```

Comment:
name seems bit misleading. should be get_tsql_const_collation or similar

Reply:
done